spec/mongoid/railties/bson_object_id_serializer_spec.rb

```diff
@@ -2,11 +2,14 @@
 # rubocop:todo all
 
 require 'spec_helper'
-require 'active_job'
-require 'mongoid/railties/bson_object_id_serializer'
 
-describe Mongoid::Railties::ActiveJobSerializers::BsonObjectIdSerializer do
-  let(:serializer) { described_class.instance }
+describe 'Mongoid::Railties::ActiveJobSerializers::BsonObjectIdSerializer' do
+  min_rails_version '6.0'
+
+  require 'active_job'
+  require 'mongoid/railties/bson_object_id_serializer'
```

Comment:
it looks like Rails 5.2 specs are still choking on the `ActiveJob::Serializers` reference. I wonder if these require statements need to be in a `before` block? Perhaps at least the serializer one?

Reply:
Since we are dropping support for Rails 5.2, we should probably just remove corresponding configuration.